jollyday-core/src/main/java/de/focus_shift/jollyday/core/HolidayCalendar.java

```diff
@@ -17,7 +17,7 @@ public enum HolidayCalendar {
   DENMARK("DK"), DOW_JONES_STOXX("DJ_STOXX"),
   ECUADOR("EC"), EGYPT("EG"), ESTONIA("EE"), ETHIOPIA("ET"),
   FINLAND("FI"), FRANCE(Locale.FRANCE.getCountry()),
-  GEORGIA("GE"), GERMANY(Locale.GERMANY.getCountry()), GUERNSEY("GG"), GREECE("GR"),
+  GIBRALTAR("GI"), GEORGIA("GE"), GERMANY(Locale.GERMANY.getCountry()), GUERNSEY("GG"), GREECE("GR"),
```

Comment:
I think the list is (mostly) alphabetical.

```suggestion
  GEORGIA("GE"), GERMANY(Locale.GERMANY.getCountry()), GIBRALTAR("GI"), GUERNSEY("GG"), GREECE("GR"),
```